yabt/__init__.py

```diff
@@ -23,5 +23,5 @@
 
 
 __author__ = 'Itamar Ostricher'
-__version__ = '0.3.15'
+__version__ = '0.3.16'
```

Comment:
Pretty sure "make release" should handle this change

Reply:
I think travis is stuck because of this. Please revert and push so it will run the actual tests...

Reply:
done